farmhash/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+
```

Comment:
Since we add this as submodule, we may remove the source code.

Reply:
now we directly include the required source code of farmhash in our code base